packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -149,7 +152,7 @@ function* loadEngineCategoriesSaga(widgetId, mediaId, callback = noop) {
         return;
       }
       if (category.engines.some(engine => engine.status !== 'complete')) {
-        category.status = 'in_progress';
+        category.status = 'inprogres';
```

Comment:
is this a typo?

Reply:
I think I saw tried status without _ in some documentation, however, couldn't find that documentation.